src/core/toga/app.py

```diff
@@ -227,6 +230,9 @@ def __init__(
         self._impl = self._create_impl()
         self.on_exit = on_exit
 
+        if default_about_command:
+            self.commands.add(AboutCommand(self))
```

Comment:
This hard-codes the about dialog that will be used. While the default will definitely be sufficient for many purposes, there will always be a call for customisation.

I'd suggest that `about()` should be a method on the app interface, with a default implementation that creates a default about dialog. While we're in the area, we should probably also factor out the other commands into similar utility methods (preferences, homepage etc) - even if they're no-ops for now.

Making `about()` an app-level method that can be overwritten also means we can remove the `default_about_command` option - a Toga will get a reasonable default, and if you want to override, that's easy enough to acheive with an override.